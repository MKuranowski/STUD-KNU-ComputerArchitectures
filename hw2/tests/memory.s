addi x20, x0, -1
addi x21, x0, 42
addi x25, x0, 84
sw x25, x20, 0
sw x25, x21, 4
sw x25, x21, -4
lw x1, x25, 0
lw x2, x25, 4
lw x3, x25, -4
