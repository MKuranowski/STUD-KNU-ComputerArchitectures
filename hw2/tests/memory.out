Processor's clock cycles: 9
>>>>>>>>[REGISTER DUMP]<<<<<<<
PC: = 36
x00 = 0
x01 = -1
x02 = 42
x03 = 42
x04 = 0
x05 = 0
x06 = 0
x07 = 0
x08 = 0
x09 = 0
x10 = 0
x11 = 0
x12 = 0
x13 = 0
x14 = 0
x15 = 0
x16 = 0
x17 = 0
x18 = 0
x19 = 0
x20 = -1
x21 = 42
x22 = 0
x23 = 0
x24 = 0
x25 = 84
x26 = 0
x27 = 0
x28 = 0
x29 = 0
x30 = 0
x31 = 0
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
>>>>>>>>[MEMORY DUMP]<<<<<<<<<
50 : 42
51 : 0
52 : 0
53 : 0
54 : 255
55 : 255
56 : 255
57 : 255
58 : 42
59 : 0
5a : 0
5b : 0
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
